### PR TITLE
make service name configurable

### PR DESCRIPTION
**Pull request by florindragos**

### CODE REVIEW

File: bosh-release/jobs/autoscaler_servicebroker/spec [hunk 1]
```diff
@@ -24,8 +24,10 @@ properties:
     description: autoscaler_server password
   autoscaler_servicebroker.auth.username:
     description: autoscaler_server username
-  autoscaler.autoscaler_servicebroker.host:
+  autoscaler_servicebroker.host:
     description: the host for the autoscaler_servicebroker
+  autoscaler_servicebroker.name:
+    description: the autoscaler service name
```
Comment (mook-as) [inline bosh-release/jobs/autoscaler_servicebroker/spec:30]: The new properties could probably use default values, so anybody who already have this (which _should_ be nobody) wouldn't need to change anything?

Comment (florindragos) [inline bosh-release/jobs/autoscaler_servicebroker/spec:30]: yeah, makes sense. I'll add "CF-AutoSclaler" as default

**Comment by mook-as** [comment]
:+1:

File: bosh-release/jobs/autoscaler_servicebroker/spec (hunk 1 above)
Comment (ericpromislow) [inline bosh-release/jobs/autoscaler_servicebroker/spec:30]: You mean "CF-AutoScaler" as default, right?

Comment (florindragos) [inline bosh-release/jobs/autoscaler_servicebroker/spec:30]: yep. was just a typo in the comment

**Pull request merged**